packaging/docker/README.md

```diff
@@ -87,8 +93,6 @@ volumes:
   netdatacache:
```

Comment:
```yaml
...
volumes:
  netdatalib:
  netdatacache:
```

=>

```yaml
volumes:
  netdataconfig:
  netdatalib:
  netdatacache:
```

Reply:
Good catch, thank you very much! Just added this.